### PR TITLE
Copy updates for np beta flow

### PR DESCRIPTION
**Pull request by aitorvs**
Task/Issue URL: https://app.asana.com/0/488551667048375/1205145153997777/f

### Description
Design/copy updates

### Steps to test this PR
QA optional

### CODE REVIEW

**Comment by aitorvs** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #3431** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/3431" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/duckduckgo/Android/3431?utm_source=stack-comment).

File: network-protection/network-protection-impl/src/main/res/drawable/ilustration_success.xml
Comment (malmstein) [inline network-protection/network-protection-impl/src/main/res/drawable/ilustration_success.xml:1]: can you rename this to `ic_illustration_success` to match the naming of other drawables?

File: network-protection/network-protection-impl/src/main/res/drawable/ilustration_network_protetion_vpn.xml
Comment (malmstein) [inline network-protection/network-protection-impl/src/main/res/drawable/ilustration_network_protetion_vpn.xml:1]: same here, can you rename this to `ic_network_protection_vpn`?

**Review by malmstein — APPROVED** [review]
couple naming nits, otherwise LGTM

File: network-protection/network-protection-impl/src/main/res/values/donottranslate.xml [hunk 1]
```diff
@@ -92,21 +92,21 @@
...
     <string name="netpWaitlistDescriptionItemTitle_3">Get your invite</string>
     <string name="netpWaitlistDescriptionItemByline_3">We can notify you when its your turn.</string>
 
-    <string name="netpWaitlistInvitedDescriptionItemTitle_1">Everyday protection for everything you do online</string>
-    <string name="netpWaitlistInvitedDescriptionItemByline_1">Network Protection uses the WireGuard protocol to secure all your network activity across your device, protecting your browsing, app activity and beyond.</string>
-    <string name="netpWaitlistInvitedDescriptionItemTitle_2">Fast, reliable and easy to use</string>
-    <string name="netpWaitlistInvitedDescriptionItemByline_2">Network Protection comes built-in to the DuckDuckGo browser. Connect to the nearest with one click.</string>
-    <string name="netpWaitlistInvitedDescriptionItemTitle_3">No logging y default</string>
-    <string name="netpWaitlistInvitedDescriptionItemByline_3">Unlike other VPNs, we do not store logs of your data. It\'s in our Privacy Policy.</string>
+    <string name="netpWaitlistInvitedDescriptionItemTitle_1">Secure all traffic to and from your device</string>
```
Comment (karlenDimla) [inline network-protection/network-protection-impl/src/main/res/values/donottranslate.xml:104]: I thought we are moving away from using "all traffic"? Might be worth confirming?

File: network-protection/network-protection-impl/src/main/res/values/donottranslate.xml [hunk 2]
```diff
@@ -92,21 +92,21 @@
...
+    <string name="netpWaitlistInvitedDescriptionItemByline_1">We use the WireGuard protocol to encrypt all your online traffic across your browsers and apps.</string>
```
Comment (karlenDimla) [inline network-protection/network-protection-impl/src/main/res/values/donottranslate.xml:105]: same here

File: network-protection/network-protection-impl/src/main/res/values/donottranslate.xml [hunk 3]
```diff
@@ -92,21 +92,21 @@
...
+    <string name="netpWaitlistInvitedDescriptionItemTitle_2">Fast, reliable, and easy to use</string>
+    <string name="netpWaitlistInvitedDescriptionItemByline_2">Connect with one click to route all your browsing and app activity through the nearest VPN server.</string>
```
Comment (karlenDimla) [inline network-protection/network-protection-impl/src/main/res/values/donottranslate.xml:107]: Here too.

**Review by karlenDimla — APPROVED** [review]
Looks good. Just curious about the copy itself (see comments).

File: network-protection/network-protection-impl/src/main/res/values/donottranslate.xml (hunk 1 above)
Comment (aitorvs) [reply network-protection/network-protection-impl/src/main/res/values/donottranslate.xml:104]: The (final) copy is in [here](https://www.figma.com/file/GWAkYzLsFuglqHxnfwYGaj/NetP-Waitlist-Flow?type=design&node-id=0-1&mode=design&t=NLXaT5q0KoxnhOfc-0).
Will double check anyways in the ship review

File: network-protection/network-protection-impl/src/main/res/values/donottranslate.xml (hunk 2 above)
Comment (aitorvs) [reply network-protection/network-protection-impl/src/main/res/values/donottranslate.xml:105]: The (final) copy is in [here](https://www.figma.com/file/GWAkYzLsFuglqHxnfwYGaj/NetP-Waitlist-Flow?type=design&node-id=0-1&mode=design&t=NLXaT5q0KoxnhOfc-0).
Will double check anyways in the ship review

File: network-protection/network-protection-impl/src/main/res/values/donottranslate.xml (hunk 3 above)
Comment (aitorvs) [reply network-protection/network-protection-impl/src/main/res/values/donottranslate.xml:107]: The (final) copy is in [here](https://www.figma.com/file/GWAkYzLsFuglqHxnfwYGaj/NetP-Waitlist-Flow?type=design&node-id=0-1&mode=design&t=NLXaT5q0KoxnhOfc-0).
Will double check anyways in the ship review

**Pull request merged**